### DISCUSSION TITLE
Jordan rate my charts on scales of 1 to 10 (marastrizs first video, chaotic fish second video)

### DISCUSSION THREAD

**Discussion by stripes10122** [JordanSantiagoYT/FNF-JS-Engine#709] [Show and tell]
[marastrizs](https://www.youtube.com/watch?v=5MM-q4Wnkxs&t)
[chaotic fish](https://www.youtube.com/watch?v=zBs8bYUNw2Q)

**Comment by JordanSantiagoYT** [comment]
marastrizs is a 6/10
Chaotic Fish is a 5.75/10

no offense, a portion of them are just offset and both songs are kind of bad

**Reply by stripes10122** [reply]
ye, i made the songs, and my pc is to bad for major spam

**Reply by stripes10122** [reply]
you can rechart them ,im uploading marastrizs +flp si can you rechart it

**Reply by stripes10122** [reply]
https://youtu.be/rq0_kCGppPI
